packages/demo/src/plasma/routes/feedback-and-info/pages/PopoverExamples.tsx

```diff
@@ -0,0 +1,84 @@
+import {loremIpsum} from 'lorem-ipsum';
+import * as React from 'react';
+import {Button, PopoverConnected, SplitLayout} from 'react-vapor';
+
+import {ExampleComponent} from '../../../utils/ExamplesUtils';
+import {useMarkdown} from '../../../../demo-building-blocs/useMarkdown';
+import VaporComponent from '../../../../demo-building-blocs/VaporComponent';
+
+const leftContent = (
+    <div>
+        <h2 className="h2">Popover Title Lorem</h2>
+        <p className="h3 pt2 pb2">I am popping under the button. {loremIpsum({count: 5, sentenceUpperBound: 15})}</p>
+    </div>
+);
+const rightContent = (
+    <div
+        className="video"
+        style={{
+            position: 'relative',
+            paddingBottom: '56.25%',
+            height: 0,
+        }}
```

Comment:
Shouldn't this style be in vapor?

Reply:
For this PR I will leave this in. Like Ariane said, this PR is meant to be moving things around